monai/apps/pathology/transforms/spatial/array.py

```diff
@@ -101,6 +134,8 @@ class TileOnGrid(Randomizable, Transform):
 
     """
 
+    backend = [TransformBackends.NUMPY]
```

Comment:
For the transform only supports `NUMPY` backend, we should add the logic to support Tensor input, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/array.py#L2038
And:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/array.py#L2045

And please also add unit tests to cover the Tensor input.

Thanks.

Reply:
This is related to `TileOnGrid` and since this PR is related to `SplitOnGrid` and to follow the principal of separation of concerns, I will address this separately. https://github.com/Project-MONAI/MONAI/issues/3379